.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+* @jjm @StackStorm-Exchange/tsc
```

Comment:
Looks like also @StackStorm-Exchange/pulsant group has write access to this repo with members @jjm, @NotRossKemp, @paul-mulvihill.
Do you know if others still around and interested in maintaining the repository?

Reply:
@jjm ah, you just covered that question as I wrote it :)

Reply:
I'm still in touch with both of them, I've pinged them a message via Signal. I think we can say @NotRossKemp is unlikely to be doing much with StackStorm, but @paul-mulvihill may be.

I think it may be best to setup a new team for the packs I still look after and create a new one for @paul-mulvihill if he wishes to continue. Also need to review the packs. As I'll not be able to contribute to hpe_icsp / vsphere packs.

Reply:
I don't mind a new group for the stuff I've done unless someone else is wanting to take ownership. I don't get to play with this anywhere as much as i'd like.
but i am trying to get spark st2 here so who knows.

Reply:
Shall we discuss which of the old Pulsant packages we each want to carry on working on off line? I'll ping you both on ST2 slack.

Reply:
Thanks for organizing this :heart: